### ISSUE TITLE
Option for creating vertical whitespace per column of pages

### ISSUE THREAD

**Issue by tipptop3d** [flxzt/rnote#878]
Currently the vertical whitespace tool moves everything below the line, but i can imagine that sometimes, if you write over multiple columns, you would only want to adjust the current column.

**Comment by flxzt** [comment]
to clarify - you're talking about selecting two vertical positions and then moving only the content between them up and down?

**Comment by tipptop3d** [comment]
Not really. I like your "semi infinite" layout to split my work horizontally onto multiple "columns", as i still need to export them to a pdf.  Maybe my sketch helps ;D I mean only moving objects that are below the red line, not over the complete width, constrained by the format borders.
![rnote](https://github.com/flxzt/rnote/assets/45404249/82c27b8d-f8a6-4992-8d2f-6b5845617bef)

**Comment by flxzt** [comment]
ahh I see. That would be quite useful indeed, I agree.

**Comment by tipptop3d** [comment]
I thought more of a checkbox or different mode under the tools menu, that if selected moves only a single column, but yes, your solution would allow a more flexible/wider selection. I would also be fine with that.

**Comment by Doublonmousse** [comment]
Somehow similar to #1048

**Comment by Doublonmousse** [comment]
I was thinking of always displaying the borders (with some external padding) and clicking on that padding would change the spacing, whilst clicking between the borders would activate the tool.

I'd say changing for this tool should display the borders in a way where they're visible and clicking again should reset them to make the borders be in view again.

**Comment by Doublonmousse** [comment]
Partially fixed by #1116.

The borders are the page borders, there isn't a more flexible way to choose the borders
